ios/ElectronCash/electroncash_gui/ios_native/wallets.py

```diff
@@ -607,12 +609,12 @@ def DeleteChk() -> None:
                     except:
                         parent.show_error(vc = vc, message = str(sys.exc_info()[1]))
                 else:
-                    parent.show_error(vc = vc, title = _("Not Deleted"), message = _("You didn't enter the text 'delete' in the previous dialog. For your own safety, the wallet file was not deleted."))
+                    parent.show_error(vc = vc, title = _("Not Deleted"), message = _("You didn't enter the text '{delete_confirm_text}' in the previous dialog. For your own safety, the wallet file was not deleted.").format(**locals()))
             Release()
-        placeholder = _("Type 'delete' to proceed")
+        placeholder = _("Type '{delete_confirm_text}' to proceed").format(**locals())
```

Comment:
I'm not sure I'm a huge fan of this approach. Would you mind changing this back to just `delete_confirm_text = delete_confirm_text`. Sorry -- I'm a paranoid person and the idea that a translation might slip through the cracks and expose internal variables bothers me in principle.